nixos/modules/virtualisation/nixos-containers.nix

```diff
@@ -499,6 +518,7 @@ let
     hostAddress6 = null;
     localAddress = null;
     localAddress6 = null;
+    macAddress = null;
```

Comment:
The host-side of the interface has his own unique MAC address as well, one day we might want to add the possibility to customize it as well.

Have you considered naming this option `localMacAddress`, following the pattern of IP addresses?